test/SolverInterface.cpp

```diff
@@ -350,6 +353,20 @@ void SolverInterface:: readScalarData
     value = fake_read_write_buffer[0]; 
 }
 
+void SolverInterface:: setMeshAccessRegion
+(
+  int meshID,
+  const double* boundingBox ) const
+{}
```

Comment:
You could implement a test, where you input a bounding box (e.g. `[1,2,3,4,5,6]`) via the python bindings and ensure here (on the cpp side) that the bounding box is received correctly. 

Same for the `meshID`: Input `0` on the python side and ensure that `0` also arrives here.